libs/librepcb/common/attributes/attributetype.cpp

```diff
@@ -69,6 +69,15 @@ const AttributeUnit* AttributeType::getUnitFromString(const QString& unit) const
         QString(tr("Unknown unit of attribute type \"%1\": \"%2\"")).arg(mTypeName, unit));
 }
 
+bool AttributeType::isUnitAvailable(const AttributeUnit* unit) const noexcept
+{
+    if (mAvailableUnits.isEmpty()) {
+        return (unit == nullptr);
+    } else {
+        return mAvailableUnits.contains(unit);
+    }
```

Comment:
The following would be clearer for me:
```C++
return (mAvailableUnits.isEmpty() && (unit == nullptr)) || mAvailableUnits.contains(unit);
```
But it would change the behavior slightly. Also maybe explain *why* it return true when the list is empty and the Unit is `nullptr`.